python/pyspark/sql/functions.py

```diff
@@ -208,6 +209,7 @@ def col(col: str) -> Column:
 column = col
 
 
+@try_remote_functions
 def asc(col: "ColumnOrName") -> Column:
```

Comment:
It seems that `try_remote_functions` needs to support this conversion `pyspark.sql.column.Column` -> `pyspark.sql.connect.column.Column`

Reply:
That's a very good point. Thanks

Reply:
Actually we don't need. It uses `pyspark.sql.connect.column.Column` column but uses `pyspark.sql.column.Column` type annotation assuming that the API coverage is same. It is same for `SparkSession` too.